src/zenml/zen_stores/migrations/versions/7e4a481d17f7_add_identity_table.py

```diff
@@ -0,0 +1,51 @@
+"""Add identity table [7e4a481d17f7].
+
+Revision ID: 7e4a481d17f7
+Revises: 7834208cc3f6
+Create Date: 2022-12-08 16:27:48.909015
+
+"""
+import sqlalchemy as sa
+import sqlmodel
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "7e4a481d17f7"
+down_revision = "7834208cc3f6"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    """Upgrade database schema and/or data, creating a new revision."""
+    from zenml.config.global_config import GlobalConfiguration
```

Comment:
This is somewhat dangerous since it makes the migration dependent on other modules, i.e., after we add this migration we will need to retroactively modify it every time we rename/delete/modify `GlobalConfiguration().user_id`.

However, the only other solution I see is to simply reset all server IDs, which we probably want even less. So I guess there is no way around importing this here.

Reply:
That's a very good point. I don't know of a better way to do this either. Perhaps we can remove the user_id from the global configuration in a future release and just populate the database with a random ID here in its place.